vita3k/renderer/src/texture/pvrt-dec.cpp

```diff
@@ -400,7 +400,7 @@ static int32_t getModulationValues(int32_t i32ModulationValues[16][8], int32_t i
                 return RepVals0[i32ModulationValues[xPos][yPos]] + modulationFlag;
             }
 
-            // else average from the neighbours
+            // else average from the neighbors
```

Comment:
This sentence comes from the powervr sdk and is valid american english, no need to change it.

Reply:
Done